webapi/_test_data/other.v1.DrawingsGodsV1Spec/drawings-gods_data.ttl~HEAD

```diff
@@ -1,5062 +0,0 @@
-@prefix owl: <http://www.w3.org/2002/07/owl#> .
```

Comment:
I assume these were leftovers of a conflict resolution

Reply:
I guess so, I didn't know where they came from.

Reply:
it comes from https://github.com/dhlab-basel/Knora/commit/adf56aa8bd74c1ee03ee77952c6baa407cfc9950#diff-d68a5b26e6169854067a9fb0e4f5d4ff

Reply:
I guess it was added bey mistake, so good that you deleted them